packages/jest-worker/src/__tests__/worker.test.js

```diff
@@ -23,16 +23,17 @@ let forkInterface;
 let childProcess;
 
 beforeEach(() => {
-  forkInterface = Object.assign(new EventEmitter(), {
-    send: jest.fn(),
-    stderr: {},
-    stdout: {},
-  });
-
   jest.mock('child_process');
 
   childProcess = require('child_process');
-  childProcess.fork.mockImplementation(() => forkInterface);
+  childProcess.fork.mockImplementation(
+    () =>
+      (forkInterface = Object.assign(new EventEmitter(), {
```

Comment:
can I buy curly braces and a few lines of code to make this more readable, please? :)

Reply:
Give thanks to prettier which made it unreadable :)